rubin_scheduler/scheduler/model_observatory/model_observatory.py

```diff
@@ -0,0 +1,631 @@
+__all__ = ("ModelObservatory", "NoClouds", "NominalSeeing")
```

Comment:
Might as well  break this part  now too and remove the “backwards compatibility” for NoClouds and NominalSeeing, since it’d be broken by the rubin-scheduler change anyway.